.travis.yml

```diff
@@ -75,9 +75,10 @@ matrix:
     - python: 2.7
     - python: 3.6
 before_install:
-  - wget -O /tmp/geckodriver.tar.gz https://github.com/mozilla/geckodriver/releases/download/v0.13.0/geckodriver-v0.13.0-linux64.tar.gz
-  - mkdir $HOME/geckodriver && tar xvf /tmp/geckodriver.tar.gz -C $HOME/geckodriver
-  - export PATH=$HOME/geckodriver:$PATH
+  - export GECKODRIVER_DOWNLOAD=`curl -s 'https://api.github.com/repos/mozilla/geckodriver/releases/latest' | python -c "import sys, json; r = json.load(sys.stdin); print([a for a in r['assets'] if 'linux64' in a['name']][0]['browser_download_url']);"`
+  - curl -L -o geckodriver.tar.gz $GECKODRIVER_DOWNLOAD
+  - gunzip -c geckodriver.tar.gz | tar xopf -
+  - chmod +x geckodriver && sudo mv geckodriver /usr/local/bin
```

Comment:
We should avoid introducing sudo.. let's just extract to a subdirectory and add that to the path, like we did previously.